opentech/apply/funds/models/applications.py

```diff
@@ -280,9 +281,59 @@ def clean(self):
 
                 raise ValidationError(error)
 
-    def serve(self, request):
+    def get_form_parameters(self, request=None):
+        form_parameters = {}
+        initial_values = {}
+
+        if request:
+            copy_open_submission = request.GET.get('open_call_submission')
+            if copy_open_submission:
+                submission_class = self.get_submission_class()
+                try:
+                    submission = submission_class.objects.get(id=copy_open_submission)
+                    if self.get_parent().id == submission.page.id:
```

Comment:
This checks that the open_call_submission is the the same fund but we also need to make sure that  open_call_submission is in state "open call".

If not there is a potential information leak.

Reply:
Thanks for noticing. Checking open call was on my list but got skipped. PR updated.